workflow.yml

```diff
@@ -29,3 +29,18 @@ jobs:
         uses: github/codeql-action/upload-sarif@v1
         with:
           sarif_file: 'trivy-results.sarif'
+
+      - name: Run Trivy vulnerability scanner on file system
+        uses: aquasecurity/trivy-action@master
+        with:
+          source-type: fs
+          image-ref: '.'
```

Comment:
I see you tried to reuse an existing flag so its backwards compatible but at the same time `image-ref` doesn't make much sense in the context of a filesystem scan. Any thoughts?

Reply:
Agree - better have a different flag for the file type path.

Reply:
@rahul2393 can you change this to `scan-ref` instead? That way it will be consistent with `scan-type`.

Reply:
@simar7 Done